src/goom_config.h

```diff
@@ -2,10 +2,10 @@
 #define GOOM_CONFIG
 
 #ifdef MID
-#define GOOM_TEXTURE_WIDTH  256
+#define GOOM_TEXTURE_WIDTH  455
```

Comment:
heh, how quaint. okay. at least make on of them a power-of-two so the other is easily handled with padding.